index.html

```diff
@@ -40,5 +61,361 @@ <h2>
+            </p>
+            <p>
+              The <a>user agent</a> MUST NOT share the audio other than audio emitted
+              from the captured tab, and MUST NOT share audio of the entire system.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h2 id="constrainable-properties">
+          Constrainable Properties for Captured Viewport Surfaces
+        </h2>
+        <p>The constraints relevant to {{MediaDevices/getViewportMedia}} are only
+        those relevant to {{MediaDevices/getDisplayMedia()}}, as defined in
+        <a data-cite="SCREEN-CAPTURE#constrainable-properties">
+          5.4 Constrainable Properties for Captured Display Surfaces</a>.
+      </section>
+      <section data-cite="permissions">
+        <h1 id="permissions-intergration">Permissions Integration</h1>
+        <p>
+          <cite>Viewport Capture</cite> is a [=powerful feature=] which is identified by the
+          [=powerful feature/name=] "viewport-capture", requiring [=express permission=] to be used.
+        </p>
+        <p>
+          As required for integration with the [[[Permissions]]] specification, this specification defines the following:
+        </p>
+        <dl>
+          <dt>
+            [=powerful feature/permission state constraints=]
+          </dt>
+          <dd>
+            Valid values for this descriptor's <a>permission state</a> are
+            {{PermissionState/"prompt"}} and {{PermissionState/"denied"}}. The user agent MUST NOT
+            ever set this descriptor's <a>permission state</a> to {{PermissionState/"granted"}}.
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h1 id=feature-policy-integration>Permissions Policy Integration</h1>
+        <p>This specification defines a [=policy-controlled feature=]
+        identified by the string
+        <code><dfn data-lt="display-capture-feature">"viewport-capture"</dfn></code>.
+        Its [=default allowlist=] is <code>"self"</code>.
+        <div class="note">
+          <p>A [=document=]'s
+          [=Document/permissions policy=]
+          determines whether any content in that document is allowed to use
+          {{MediaDevices/getViewportMedia}}. If disabled in any document, no content in
+          the document will be
+          [=allowed to use=]
+          {{MediaDevices/getViewportMedia}}. This is enforced by the
+          [=prompt the user to choose=] algorithm.
+          </p>
+        </div>
+      </section>
+    </section>
+    <section>
+      <h1>Privacy Indicator Requirements</h1>
+      <p>This specification extends the <a data-cite="SCREEN-CAPTURE#privacy-indicator-requirements">
+        Privacy Indicator Requirements</a> of
+      {{MediaDevices/getDisplayMedia()}} to include {{MediaDevices/getViewportMedia}}.</p>
+    </section>
+    <section>
+      <h2>
+        Security and Permissions
```

Comment:
```suggestion
        Security considerations
```

Reply:
Shouldn't we be consistent with:
- [Security and Permissions](https://w3c.github.io/mediacapture-screen-share/#security-and-permissions) in getDisplayMedia?

?

Both APIs integrate with permissions

Reply:
On re-reading [Security and Permissions](https://w3c.github.io/mediacapture-screen-share/#security-and-permissions) in getDisplayMedia, there's not a lot of mention of permissions, so I'm accepting this edit.